hack/install-kubestellar.sh

```diff
@@ -121,26 +125,30 @@ else
     echo "Specified folder does not exist: $kubestellar_folder" >&2; exit 1;
 fi
 
-if [ $verbose == "true" ]
-then
-    echo "Downloading Kubestellar $kubestellar_version $kubestellar_os/$kubestellar_arch..."
+if [ $verbose == "true" ]; then
+    echo "Downloading KubeStellar $kubestellar_version $kubestellar_os/$kubestellar_arch..."
     curl -SL -o kubestellar.tar.gz "https://github.com/kcp-dev/edge-mc/releases/download/${kubestellar_version}/kcp-edge_${kubestellar_version}_${kubestellar_os}_$kubestellar_arch.tar.gz"
 else
     curl -sSL -o kubestellar.tar.gz "https://github.com/kcp-dev/edge-mc/releases/download/${kubestellar_version}/kcp-edge_${kubestellar_version}_${kubestellar_os}_$kubestellar_arch.tar.gz"
 fi
 
-if [ $verbose == "true" ]
-then
+if [ $verbose == "true" ]; then
     echo "Extracting archive to: $kubestellar_folder"
 fi
-tar -zxf kubestellar.tar.gz -C $kubestellar_folder
+if [ $kubestellar_strip_bin == "true" ]; then
+    tar -C $kubestellar_folder -zxf kubestellar.tar.gz --wildcards --strip-components=1 bin/*
+    bin_folder=$(get_full_path "$kubestellar_folder")
+else
+    tar -C $kubestellar_folder -zxf kubestellar.tar.gz
+    bin_folder=$(get_full_path "$kubestellar_folder/bin")
+fi
 
-if [ $verbose == "true" ]
-then
+if [ $verbose == "true" ]; then
     echo "Cleaning up..."
```

Comment:
Considering that we use the wording "cleanup" for a different activity later, maybe different words would be better here?  Something like "removing temporary files" or "removing downloaded archive"?

Reply:
ok